build/NuSpecs/WindowsAppSDK-Nuget-Native.CompatibilitySetter.targets

```
@@ -0,0 +1,73 @@
+    <PropertyGroup>
+      <WinAppSDKCompatibilitySetterFile>$(GeneratedFilesDir)WinAppSDKCompatibilitySetter.cpp</WinAppSDKCompatibilitySetterFile>
+      <WinAppSDKCompatibilityPatchMode1Lines Condition="'$(WindowsAppSDKRuntimePatchMode)' != ''">
+            compatibilityOptions.PatchMode1({$(WindowsAppSDKRuntimePatchMode.Replace(".", ","))})%3B
+      </WinAppSDKCompatibilityPatchMode1Lines>
+      <WinAppSDKCompatibilityPatchMode2Lines Condition="'$(WindowsAppSDKRuntimePatchMode2)' != ''">
+            compatibilityOptions.PatchMode2({$(WindowsAppSDKRuntimePatchMode2.Replace(".", ","))})%3B
+      </WinAppSDKCompatibilityPatchMode2Lines>
+      <WinAppSDKCompatibilityDisabledChangesNoSpaces Condition="'$(WindowsAppSDKDisabledChanges)' != ''">$([System.Text.RegularExpressions.Regex]::Replace($(WindowsAppSDKDisabledChanges), "\s+", ""))</WinAppSDKCompatibilityDisabledChangesNoSpaces>
+      <WinAppSDKCompatibilityDisabledChangesLines Condition="'$(WinAppSDKCompatibilityDisabledChangesNoSpaces)' != ''">
+            CompatibilityChange disabledChangesArray[] = { CompatibilityChange::$([System.Text.RegularExpressions.Regex]::Replace($(WinAppSDKCompatibilityDisabledChangesNoSpaces), ",([A-Za-z])", ", CompatibilityChange::$1")) }%3B
+            for (auto changeId : disabledChangesArray)
+            {
+                compatibilityOptions.DisabledChanges().Append(changeId)%3B
+            }
+      </WinAppSDKCompatibilityDisabledChangesLines>
+      
+      <WinAppSDKCompatibilitySetterFileLines>
+// This file is generated by the build based on project properties.
+#include &lt;winrt/Windows.Foundation.h&gt;
+#include &lt;winrt/Windows.Foundation.Collections.h&gt;
+
+#include &lt;winrt/Microsoft.Windows.ApplicationModel.WindowsAppRuntime.h&gt;
+
+using namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime%3B
+
+namespace Microsoft::Windows::ApplicationModel::WindowsAppRuntime::Compatibility
+{
+    namespace AutoInitialize
+    {
+        // Called by WindowsAppRuntimeAutoInitializer.cpp
+        void Initialize()
+        {
+            CompatibilityOptions compatibilityOptions%3B
+$(WinAppSDKCompatibilityPatchMode1Lines)
+$(WinAppSDKCompatibilityPatchMode2Lines)
+$(WinAppSDKCompatibilityDisabledChangesLines)
+            compatibilityOptions.Apply()%3B
+        }
+    }
+}
+      </WinAppSDKCompatibilitySetterFileLines>
+    </PropertyGroup>
+
+    <WriteLinesToFile
+        File="$(WinAppSDKCompatibilitySetterFile)" Lines="$(WinAppSDKCompatibilitySetterFileLines)"
+        Overwrite="true"
+        WriteOnlyWhenDifferent="true" />
+
+    <ItemGroup>
+      <ClCompile Include="$(WinAppSDKCompatibilitySetterFile)">
+        <PrecompiledHeader>NotUsing</PrecompiledHeader>
+      </ClCompile>
+    </ItemGroup>
+  </Target>
+
+  <PropertyGroup>
+    <BeforeClCompileTargets>
+        $(BeforeClCompileTargets); WinAppSDKCompatibilitySetterTarget
```

Best to be paranoid:
```suggestion
        $(BeforeClCompileTargets); WinAppSDKCompatibilitySetterTarget
```